mongodbatlas/data_source_mongodbatlas_team_test.go

```diff
@@ -52,5 +52,10 @@ func testAccDataSourceMongoDBAtlasTeamConfig(orgID, name, username string) strin
 			org_id     = mongodbatlas_teams.test.org_id
 			team_id    = mongodbatlas_teams.test.team_id
 		}
+
+		data "mongodbatlas_teams" "test2" {
+			org_id     = mongodbatlas_teams.test.org_id
+			name    = mongodbatlas_teams.test.name
+		}
```

Comment:
I think is better to have another test case with this specific TF configuration to test separately